api_loader.go

```diff
@@ -406,8 +406,8 @@ func processSpec(spec *APISpec, apisByListen map[string]int,
 		mwAppendEnabled(&chainArray, &StripAuth{baseMid})
 		mwAppendEnabled(&chainArray, &KeyExpired{baseMid})
 		mwAppendEnabled(&chainArray, &AccessRightsCheck{baseMid})
-		mwAppendEnabled(&chainArray, &RateLimitForAPI{BaseMiddleware: baseMid})
 		mwAppendEnabled(&chainArray, &RateLimitAndQuotaCheck{baseMid})
+		mwAppendEnabled(&chainArray, &RateLimitForAPI{BaseMiddleware: baseMid})
```

Comment:
Is this related? or just testing?

Reply:
not related - was a checkout of another branch

Reply:
Can you put link to related PR, so we can merge it, and you will rebase on master.